src/components/TableFilter.js

```diff
@@ -264,6 +268,19 @@ class TableFilter extends React.Component {
     );
   }
 
+  renderCustomField(column, index) {
+    const { classes, filterList, options } = this.props;
+    const display = (column.filterOptions && column.filterOptions.display) || options.filterOptions.display;
```

Comment:
This will result in an error when the type is set to `'custom'` but there is no `display` property defined. I think we should be kind here and instead display an empty field with a console error that explains that the `display` property must be set if using a `'cusomt'` filter type.

Reply:
Added a message, feel free to tweak the wording if you want!